tests/Fixer/DoctrineAnnotation/DoctrineAnnotationBracesFixerTest.php

```diff
@@ -39,7 +39,18 @@ public function testFixWithBraces($expected, $input = null)
      */
     public function provideFixWithBracesCases()
     {
-        return $this->createTestCases(array(
+        $edgeCases = array(
```

Comment:
Without the fix, does this edge case also fail with other Doctrine Annotation fixers? If so, please add it to related test classes as well.

Reply:
Yes, but to be fair there should be an `AbstractDoctrineAnnotationFixerTest`, it's not a good thing to have repeated test cases all over subclasses.

Reply:
I would still add the case in existing test classes. @keradus @SpacePossum WDYT?

Reply:
I agree with both of you :)
If there is an `AbstractDoctrineAnnotationFixer` than there should be a `AbstractDoctrineAnnotationFixerTest` (there should be, maybe introduce it here? not sure if it is a lot of work), if there is not than the test should be added to the utests of all of the effected children

Reply:
Done